version/version.go

```diff
@@ -0,0 +1,44 @@
+/***
+Copyright 2014 Cisco Systems Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manage versioning for netmaster/netplugin apis
+
+package version
+
+import "fmt"
+
+// Info enlists version and build information as used by netplugin binaries
+type Info struct {
+	GitCommit string
+	Version   string
+	BuildTime string
+}
+
+// Get gets the version information
+func Get() *Info {
+	ver := Info{}
+	ver.GitCommit = gitCommit
+	ver.Version = versionStr
+	ver.BuildTime = buildTime
+
+	return &ver
+}
+
+// Print prints the versioning details
+func Print(ver *Info) {
```

Comment:
If you replace this with:

```
func (i *Info) String() {
   // returns a string instead
}
```

any print of `i` (e.g., `fmt.Println` or `fmt.Printf("%#v")`) will yield this data. I would prefer this as it's more flexible (and not a huge change from what we have here)